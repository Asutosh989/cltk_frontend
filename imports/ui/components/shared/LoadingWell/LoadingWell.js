

LoadingWell = React.createClass({

	render() {
		return (
			<div className="well-spinner-double">
				<div className="double-bounce1" />
				<div className="double-bounce2" />
			</div>

		);
	},

});
