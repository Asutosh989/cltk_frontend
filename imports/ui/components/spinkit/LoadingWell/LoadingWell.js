
import React from 'react';

export default class LoadingWell extends React.Component {
	render() {
		return (
			<div className="loading-spinner loading-spinner-well">
			</div>
		);
	}
}
