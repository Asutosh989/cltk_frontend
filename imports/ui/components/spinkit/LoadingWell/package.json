{
  "name": "LoadingWell",
  "version": "0.0.1",
  "private": true,
  "main": "./LoadingWell.js"
}
