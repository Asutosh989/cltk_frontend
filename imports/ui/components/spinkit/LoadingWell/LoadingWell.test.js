
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import LoadingWell from './LoadingWell';

describe('LoadingWell', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<LoadingWell />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
