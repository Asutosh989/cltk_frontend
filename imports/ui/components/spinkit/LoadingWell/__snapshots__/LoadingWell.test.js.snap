// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingWell renders correctly 1`] = `
<div
  className="loading-spinner loading-spinner-well"
/>
`;
